### ISSUE TITLE
Support for API v3

### ISSUE THREAD

**Issue by innerjoin** [jazz-community/rtc-git-commit-picker#4]
The current implementation is based on **API v4**, which was introduced with GitLab 9.0. 
The integration should detect the GitLab version and then decide to which API endpoint it should connect. 
Furthermore, GitLab releases prior to 8.9 do not support *Personal Access Tokens*, so it should allow to login with username/password.

**Comment by innerjoin** [comment]
implementation currently blocked by #2

**Comment by innerjoin** [comment]
**API v3 vs v4**
More information is available in the [API V3 to API V4 ](https://docs.gitlab.com/ce/api/v3_to_v4.html) guide

**Comment by innerjoin** [comment]
This project has been phased out a while ago. Anyone interested in integrating RTC with GitLab or GitHub should check out the [RTC Git Connector](https://github.com/jazz-community/rtc-git-connector) project, which covers all the use cases of this repo plus many more.
If you feel like this issue is still relevant with what the [RTC Git Connector](https://github.com/jazz-community/rtc-git-connector) aims to provide, feel free to open a new issue there
